### PR TITLE
Docs for Teams Activity Feed APIs in v1.0

### PR DESCRIPTION
**Pull request by eddie-lee-msft**

### CODE REVIEW

File: api-reference/v1.0/resources/teamworkactivitytopicsource.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+localization_priority: Normal
+ms.prod: "microsoft-teams"
+doc_type: enumPageType
+---
+
+# teamworkActivityTopicSource enum type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
```
Comment (RamjotSingh) [inline api-reference/v1.0/resources/teamworkactivitytopicsource.md:14]: Remove :)

**Review by RamjotSingh — APPROVED** [review]

File: api-reference/v1.0/resources/teamworkactivitytopicsource.md (hunk 1 above)
Comment (eddie-lee-msft) [reply api-reference/v1.0/resources/teamworkactivitytopicsource.md:14]: rats, i thought I removed it from all. nice catch, ty.

File: api-reference/v1.0/api/chat-sendactivitynotification.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+## Examples
+
+### Example 1: Notify a user about a task created in a chat
+
+This example shows how you can send an activity feed notification for a new task created in a chat. For more details, see [sending Teams activity notifications](/graph/teams-send-activityfeednotifications).
+
+#### Request
+
+# [HTTP](#tab/http)
```
Comment (Lauragra) [inline api-reference/v1.0/api/chat-sendactivitynotification.md:73]: Please remove the HTTP H1; it gets added via automation when we add the snippets.

File: api-reference/v1.0/resources/teamworkactivitytopicsource.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: "teamworkActivityTopicSource enum type"
+description: "Types of teamworkActivityTopic."
+author: "eddie-lee-msft"
+localization_priority: Normal
+ms.prod: "microsoft-teams"
+doc_type: enumPageType
+---
+
+# teamworkActivityTopicSource enum type
```
Comment (Lauragra) [inline api-reference/v1.0/resources/teamworkactivitytopicsource.md:10]: We don't create individual pages for enums. Please add this to enums.md. See this guidance: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103158/Enumerations

**Review by Lauragra — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**